HTML+CSS+JS/Background/Auto-Scrolling-Background/style.css

```diff
@@ -0,0 +1,43 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+.text
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    width: 100vw;
+    z-index: 1;
+    font-size:70px;
+    font-family: cursive;
+    color: white;
+}
+.container {
+  overflow: hidden;
+  height: 100vh;
+  width: 100vw;
+  position: relative;
+}
+
+.sliding-background {
+    z-index: -1;
+  background: url("../../../assets/MUI\ Logo.png")
```

Comment:
Actually, I meant that you first add the assets folder in `HTML+CSS+JS/Background/Auto-Scrolling-Background` then add the image in `HTML+CSS+JS/Background/Auto-Scrolling-Background/assets` folder. Then use the image from there. Like this everything related to the component will remain in the same directory.
Can you please make this final change.

Reply:
Okay @theBeginner86 Will do it